.github/workflows/reusable-build-build.yml

```diff
@@ -58,7 +58,6 @@ jobs:
       - name: Trim paths
         run: |
           echo $'\n' >> .cargo/config.toml
-          echo '[unstable]' >> .cargo/config.toml
           echo 'trim-paths = true' >> .cargo/config.toml
```

Comment:
is this needed removed too?

Reply:
Yes, because it is added to `.cargo/config.toml`